helm/vitess/templates/_keyspace.tpl

```diff
@@ -0,0 +1,69 @@
+###################################
+# keyspace initializations
+###################################
+
+{{- define "keyspace" -}}
+{{- $cell := index . 0 -}}
+{{- $keyspace := index . 1 -}}
+{{- $defaultVtctlclient := index . 2 -}}
+{{- $namespace := index . 3 -}}
+
+# sanitize inputs for labels
+{{- $keyspaceClean := include "clean-label" $keyspace.name -}}
+
+{{- with $cell.vtctld -}}
+
+# define image to use
+{{- $vitessTag := .vitessTag | default $defaultVtctlclient.vitessTag -}}
+
+{{ if $keyspace.vschema }}
+{{/* force blank line */}}
```

Comment:
this line is unnecessary. Just remove the last `-` in ` $defaultVtctlclient.vitessTag -}}` so it reads `$defaultVtctlclient.vitessTag }}`